pkg/bff/auth/metadata.go

```diff
@@ -1,19 +1,47 @@
 package auth
 
-import "encoding/json"
+import (
+	"encoding/json"
+
+	"github.com/trisacrypto/directory/pkg/bff/models/v1"
+)
 
 // AppMetadata makes it easier to serialize and deserialize JSON from the auth0
 // app_metadata assigned to the user by the BFF (and ensures the data is structured).
 type AppMetadata struct {
-	OrgID string `json:"orgid"`
-	VASPs VASPs  `json:"vasps"`
+	OrgID         string              `json:"orgid"`
+	VASPs         VASPs               `json:"vasps"`
+	Organizations map[string]struct{} `json:"organizations"`
 }
 
 type VASPs struct {
 	MainNet string `json:"mainnet"`
 	TestNet string `json:"testnet"`
 }
 
+// TODO: Hash-based method might be more maintainable, but this avoids error handling for now
+func (meta *AppMetadata) Equals(other *AppMetadata) bool {
+	if meta.OrgID != other.OrgID {
+		return false
+	}
+
+	if meta.VASPs != other.VASPs {
+		return false
+	}
```

Comment:
Note that struct equality only works the way you expect here if the `VASP` struct only has comparable fields:

https://golangbyexample.com/struct-equality-golang/

If we add say a `[]string` to `VASP` then this check will no longer work.

Reply:
Right, but I think that's a compile error so at least it won't catch us by surprise.